apps/tenant-management-webapp/src/app/pages/admin/services/file/file.tsx

```diff
@@ -7,28 +7,29 @@ import { RootState } from '@store/index';
 import { Aside, Main, Page } from '@components/Html';
 import { Tab, Tabs } from '@components/Tabs';
 import SupportLinks from '@components/SupportLinks';
+import { Gapadjustment, Hyperlinkcolor } from '@pages/admin/dashboard/styled-components';
+import { ExternalLink } from '@components/icons/ExternalLink';
 
 const HelpLink = (): JSX.Element => {
   const tenantName = useSelector((state: RootState) => state.tenant?.name);
   const docBaseUrl = useSelector((state: RootState) => state.config.serviceUrls?.docServiceApiUrl);
+  function getFileDocsLink() {
+    return `${docBaseUrl}/${tenantName?.toLowerCase().replace(/ /g, '-')}?urls.primaryName=File service`;
+  }
+  function getFilesupportcodeLink() {
+    return 'https://github.com/GovAlta/adsp-monorepo/tree/main/apps/file-service';
+  }
   return (
     <>
-      <h3>Helpful links</h3>
-      <a
-        rel="noopener noreferrer"
-        target="_blank"
-        href={`${docBaseUrl}/${tenantName?.toLowerCase().replace(/ /g, '-')}?urls.primaryName=File service`}
-      >
-        Read the API docs
-      </a>
-      <br />
-      <a
-        rel="noopener noreferrer"
-        target="_blank"
-        href="https://github.com/GovAlta/adsp-monorepo/tree/main/apps/file-service"
-      >
-        See the code
-      </a>
+      <Gapadjustment>Helpful links</Gapadjustment>
```

Comment:
If this code be used in many another place, could you make component in apps/tenant-management-webapp/src/app/components.

Reply:
refactored